.github/workflows/package-Instrumentation.AWS.yml

```diff
@@ -0,0 +1,50 @@
+name: Pack OpenTelemetry.Contrib.Instrumentation.AWS
```

Comment:
@eddynaka @cijothomas Do we have a MyGet workflow for contrib? Thinking we probably want a shared one that does all the projects?

Reply:
Hi @CodeBlanch We had a conversation in the last few SIG meetings to decouple the projects since the contrib repo was blocked beause some projects were broken due to API changes in the core repo. The proposal for the same is here:  https://github.com/open-telemetry/opentelemetry-dotnet-contrib/issues/57
So we decided to have separate release workflows as well for projects in this PR: https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/64